base_rest/components/service.py

```diff
@@ -175,6 +175,23 @@ def _validator_get(self):
         """
         return {}
 
+    def _get_validator_handler(self, method_name):
```

Comment:
@guewen I would like to avoid to re introduce some logic dedicated to the message processing into  `base.rest.service`. 
Instead of adding these new methods, could we not consider to look for a component in the context of the service with usage='cerberus.validator' ? The `RestApiMethodTransmer` and `CerberusValidator`classes could lookup the registry to get this component and you could extend it into shopfloor :thinking:

Reply:
This is a better separation of concern, yes.